modules/apim_gateway/manifests/params.pp

```diff
@@ -14,95 +14,32 @@
 #  limitations under the License.
 # ----------------------------------------------------------------------------
 
-# Claas apim::params
+# Class apim::params
```

Comment:
Shouldn't this be apim_gateway params?

Reply:
Resolved https://github.com/wso2/puppet-apim/pull/86/commits/5dd25054bc378c8b256e4571b433b51eecc8d807